std/json.d

```diff
@@ -1725,8 +1727,11 @@ pure nothrow @safe unittest // issue 15884
     assert(test(30738.22));
 
     assert(test(1 + double.epsilon));
-    assert(test(-double.max));
     assert(test(double.min_normal));
+    static if (realInDoublePrecision)
+        assert(test(-double.max / 2));
+    else
+        assert(test(-double.max));
```

Comment:
This may have been a more severe issue (it's been a while since I relaxed them for LDC), I need to recheck.